config/routes.rb

```diff
@@ -1,3 +1,13 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'sessions/new'
+  root 'microposts#index'
+  get '/signup', to: 'users#new'
+  post '/signup', to: 'users#create'
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
+  get '/index', to: 'microposts#index'
+  get '/micropost', to: 'microposts#new'
+  resources :users
+  resources :microposts, only: [:create, :index, :destroy]
```

Comment:
I don't think we need `:destroy` right, i see no where you're deleting post, according to the project instructions you are to use [:new, :create, :index] right.
![image](https://user-images.githubusercontent.com/31897434/67447528-17274200-f60c-11e9-9a7f-7d9c98ac7ec4.png)